createsend-dotnet/Client.cs

```diff
@@ -31,9 +31,28 @@ public ClientWithSettings Details()
             return HttpGet<ClientWithSettings>(string.Format("/clients/{0}.json", ClientID), null);
         }
 
-        public IEnumerable<CampaignDetail> Campaigns()
+        public PagedCollection<CampaignDetail> Campaigns(
+            string tags,
+            int page,
+            int pageSize,
+            string orderDirection,
+            DateTime sentFromDate,
+            DateTime sentToDate)
         {
-            return HttpGet<CampaignDetail[]>(string.Format("/clients/{0}/campaigns.json", ClientID), null);
+            return Campaigns(tags, page, pageSize, orderDirection, sentFromDate.ToString("yyyy-MM-dd"), sentToDate.ToString("yyyy-MM-dd"));
+        }
+
+        public PagedCollection<CampaignDetail> Campaigns(string tags = null, int page = 1, int pageSize = 100, string orderDirection = "desc", string sentFromDate = null, string sentToDate = null)

```

Comment:
Consider having at least one mandatory parameter here (perhaps page?) to make it more likely the developer will notice the change of contract.